public/javascripts/SVLabel/css/svl.css

```diff
@@ -8,10 +8,10 @@
 text {
     visibility: hidden;
 }
-
+/*
```

Comment:
Thanks for the clarification. That makes sense if the visibility of stats and others are being managed in other files as well. Could you resolve the conflicts? I will merge it then.